backend/aiService.js

```diff
@@ -27,23 +27,98 @@ const aiService = {
       price,
     } = productDetails;
 
-    const { age, sex, height, weight, mobile_platform, activity_level } =
-      personDetails;
+    let parsedJson = null;
+    if (process.env.PERSONALIZATION_DISABLED != "true") {
+      const { age, sex, height, weight, mobile_platform, activity_level } =
+        personDetails;
```

Comment:
```suggestion
      const { age, sex, height, weight, mobile_platform, activity_level } =
        personDetails;
```

These variables are not being used anywhere.